plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestColumnWithAliasFormatterFixed.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import io.trino.spi.session.PropertyMetadata;
+import io.trino.testing.TestingConnectorSession;
+import org.testng.annotations.Guice;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.jdbc.JdbcMetadataSessionProperties.COLUMN_ALIAS_MAX_CHARS;
+import static io.trino.plugin.jdbc.TestingJdbcTypeHandle.JDBC_VARCHAR;
+import static io.trino.spi.session.PropertyMetadata.integerProperty;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Guice(modules = ColumnWithAliasFormatterModule.class)
+public class TestColumnWithAliasFormatterFixed
+{
+    private static final ImmutableList<PropertyMetadata<?>> PROPERTY_METADATA = ImmutableList.of(
+            integerProperty(
+                    COLUMN_ALIAS_MAX_CHARS,
+                    "Column Alias max characters",
+                    null,
+                    false));
+
+    @Inject
+    private ColumnWithAliasFormatterFixed actor;
+
+    private final TestingConnectorSession session = TestingConnectorSession.builder()
+            .setPropertyMetadata(PROPERTY_METADATA)
+            .setPropertyValues(singletonMap(COLUMN_ALIAS_MAX_CHARS, 30))
+            .build();
+
+    @Test
+    public void testTooLongName()
+    {
+        //given
```

Comment:
drop given, when, then comments

Reply:
I'm sorry that what seems like simple comments with obvious 'fixes' to apply are triggering lengthy conversations. My justification is that it's the only way I learn the sometimes unusual way things are used/not-used in trino. Here's another one that would probably exceed what everyone involved expected to happen here.

The reason why I added those comments, is because it's nowadays an industry standard to use those labels that stem from BDD (probably dated back to 200x). In fact, modern testing frameworks have actually incorporated those into the language itself, like for instance spock: https://spockframework.org/spock/docs/1.3/all_in_one.html#_blocks. In last 2 projects I've been involved with we actually used spock for testing. I must say that I was a bit surprised that the project that was advertised to me as following the latest industry standards still uses testNG.

I find it awkward that you ask me to remove the labels, which AFAIK are a way to emulate this missing BDD feature in legacy testing libraries. Can you let me know what other means you use for emulating it? If not, when do you plan upgrading to spock? If it's not former, nor the latter, can you sched some light on what is the thinking behind your request?

Reply:
Dropped.